### PR TITLE
mainnet launch revision

### PR DESCRIPTION
**Pull request by barakshani**
**NOT TO MERGE** until:
- [x] TODO's are resolved in https://github.com/matter-labs/zksync-web-v2-docs/blob/v2_oct_revision/docs/dev/developer-guides/transactions/blocks.md
- [x] 'edit' page is edited or removed https://github.com/matter-labs/zksync-web-v2-docs/blob/v2_oct_revision/docs/dev/glossary/edit-doc.md

### CODE REVIEW

File: docs/dev/developer-guides/README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ Take a deeper dive into the zkSync ecosystem.
...
-- [L1 / L2 communication](./developer-guides/bridging/l1-l2.md)
-- [L2 / L1 communication](./developer-guides/bridging/l2-l1.md)
+- [Quickstart](./hello-world.md)
+- [System contract](./contracts/system-contracts.md)
+- [Contract deployment](./contracts/contracts.md) 
+- [Contract verification](./contracts/contract-verification.md)
+- [Transaction finality](./transactions/transactions.md)
+- [Account abstraction](./transactions/aa.md)
+- [Fee mechanism](./fee-model.md)
+- [bridging of funds](./bridging/bridging-asset.md)
```
Comment (uF4No) [inline docs/dev/developer-guides/README.md:14]: Should be "Bridging" with capitals in the link

Comment (barakshani) [reply docs/dev/developer-guides/README.md:14]: looks like it got reverted somehow: https://github.com/matter-labs/zksync-web-v2-docs/pull/108/commits/c5818f563724b48e97746ed468eca93900faf98d#diff-8af04b40623f9f2c3be342ff05037fa034ca61f2cc921a61a9322823e4082d19R26
will fix it

File: docs/dev/developer-guides/contracts/system-contracts.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # Understanding system contracts
-To keep the zero-knowledge circuits as simple as possible and enable simple extensions, a large chunk of the logic of zkSync was moved to the so-called "system contracts" -- a set of contracts that have special privileges and serve special purposes, e.g. deployment of contracts, making sure that the user pays only once for publishing contracts' calldata, etc.
+To keep the zero-knowledge circuits as simple as possible and enable simple extensions. A large chunk of the logic of zkSync was moved to the so-called "system contracts" – a
```
Comment (uF4No) [inline docs/dev/developer-guides/contracts/system-contracts.md:2]: this should not be two separate phrases.

File: docs/dev/developer-guides/contracts/system-contracts.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 # Understanding system contracts
-To keep the zero-knowledge circuits as simple as possible and enable simple extensions, a large chunk of the logic of zkSync was moved to the so-called "system contracts" -- a set of contracts that have special privileges and serve special purposes, e.g. deployment of contracts, making sure that the user pays only once for publishing contracts' calldata, etc.
+To keep the zero-knowledge circuits as simple as possible and enable simple extensions. A large chunk of the logic of zkSync was moved to the so-called "system contracts" – a 
+set of contracts that have special privileges and serve special purposes, e.g. deployment of contracts, making sure that the user pays only once for publishing contracts' calldata, etc.
 
-Until the code for the system contracts has gone through thorough testing, the code for all the system contracts will not be public. This section will only provide you with the knowledge needed to build on zkSync.
+Until the code for the system contracts has gone thorough testing, the code for all the system contracts will not be public. This section will only provide you with the knowledge needed to build on zkSync.
```
Comment (uF4No) [inline docs/dev/developer-guides/contracts/system-contracts.md:5]: Keep original "through thorough testing"

File: docs/dev/developer-guides/contracts/system-contracts.md (hunk 2 above)
Comment (barakshani) [reply docs/dev/developer-guides/contracts/system-contracts.md:2]: what is "this"?

Comment (barakshani) [reply docs/dev/developer-guides/contracts/system-contracts.md:2]: if you mean the line break, it is done automatically by the ide. It displays as one sentence on html

Comment (uF4No) [reply docs/dev/developer-guides/contracts/system-contracts.md:2]: I mean, "and enable simple extensions. A large chunk of the logic" should be "and enable simple extensions, a large chunk of the logic".

File: docs/dev/developer-guides/transactions/blocks.md [hunk 4]
```diff
@@ -1,46 +1,49 @@
...
 
-A block in Ethereum is a list of transactions that have been recorded historically. They are collections of transactions that include the hash of the chain's previous block.
-We group transactions into blocks to ensure everyone on the network stays in sync and agrees on the detailed history of transactions. It implies that hundreds of transactions are simultaneously committed, decided upon, and synced.
+A block is an ordered list of transactions. Each block (except for the Genesis block) points to the previous block it extends, thus creating a chain of blocks.
 
-## Prerequisite
+## Blocks in zkSync 2.0
 
-To fully understand this page, we suggest you first read a guide on [transactions](transactions.md).
+In zkSync there are two notions of ``blocks'': an L2 block and an L1 rollup block.
```
[the remark this answers is not in the document]
Comment (barakshani) [reply docs/dev/developer-guides/transactions/blocks.md:7]: it's on purpose (it's not a quote). Yes, cause batch are not really blocks.

File: docs/dev/docs-contribution/edit-doc.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Edit or review a documentation page
+
+### Goal
+Find a documentation page that you think needs updating. proofread, or edit the page.
+
+### Skills required
+
+- Understands and writes English well
```
[the remark this answers is not in the document]
Comment (barakshani) [reply docs/dev/docs-contribution/edit-doc.md:8]: I thought I removed this -- let me check again...

File: docs/dev/developer-guides/transactions/blocks.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (uF4No) [reply docs/dev/developer-guides/transactions/blocks.md:7]: it looks weird on the page, like if it was a typo so I think we should put it on normal quotes(") or backticks (`), but the same in both ends.

**Comment by barakshani** [comment]
@uF4No any further comments? Are we good to proceed?

**Comment by uF4No** [comment]
> @uF4No any further comments? Are we good to proceed?

@barakshani Just replace "bridging" with "Bridging" in `[docs/dev/developer-guides/README.md](https://github.com/matter-labs/zksync-web-v2-docs/pull/108/files#diff-0ee300e2aaa0e1ada21ba7577e7b342e74d06ecd04d4d5cad02d374acf6af860)`

**Review by uF4No — APPROVED** [review]

**Review by uF4No — APPROVED** [review]

**Comment by uF4No** [comment]
approved and conversation resolved

**Comment by barakshani** [comment]
@StanislavBreadless please merge the PR

**Comment by barakshani** [comment]
@uF4No it looks like you also of merge rights (I don't), please merge this one

**Comment by uF4No** [comment]
merged

**Pull request merged**